provisioning catalog using csv compatible syntax (#139)
kuzzleio/kuzzle-device-manager: doc/1/controllers/device/import-catalog/index.md

---
code: true
type: page
title: importCatalog
description: Provisioning of the device catalog via csv import
---

# importCatalog

> Provisioning of the device catalog via csv import

Reviewer: ```suggestion
Creates entries in the provisioning catalog via CSV import
```
